src/main/integrations/last-fm/index.ts

```diff
@@ -97,14 +114,14 @@ export default class LastFM implements IIntegration {
 
       this.updateNowPlaying(state.videoDetails);
 
+      const scrobblePercentDecimal = this.lastfmDetails.scrobblePercent / 100;
       const scrobbleTimeRequired = Math.min(
-        // Scrobble the track if it has been played for more than 50% of its duration
-        Math.round(state.videoDetails.durationSeconds / 2),
-        // OR if it has been played for more than 4 minutes
-        240
+        // Scrobble the track if it has been played to the percent picked by the user
+        Math.round(state.videoDetails.durationSeconds * scrobblePercentDecimal),
+        // OR if it has been played for more than 5 minutes
+        300
```

Comment:
Why has this changed to 5 minutes?

As per Last.fm specs:
> "And the track has been played for at least half its duration, or for 4 minutes (whichever occurs earlier.)"
[Last.fm Document](https://www.last.fm/api/scrobbling#:~:text=And%20the%20track%20has%20been%20played%20for%20at%20least%20half%20its%20duration%2C%20or%20for%204%20minutes%20(whichever%20occurs%20earlier.))